aws/lambda_demo/lambda_function.py

```diff
@@ -0,0 +1,123 @@
+import json, os
+import sentry_sdk
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+from sentry_sdk.integrations.serverless import serverless_function
+from datetime import datetime, timezone
+from contextlib import contextmanager
+
+
+sentry_sdk.init(
+    dsn=os.environ.get("PYTHON_LAMBDA_SENTRY_DSN"),
```

Comment:
please rename directory `aws` -> `aws-lambda-python` or `python-aws-lambda` to be more clear